src/api/files/service.js

```diff
@@ -441,5 +442,5 @@ export async function checkFileStatus(fileId) {
 /**
  * @import { SubmitPayload, SubmitRecordset } from '@defra/forms-model'
  * @import { Input, Callback } from 'csv-stringify'
- * @import { FileUploadStatus, UploadPayload } from '~/src/api/types.js'
+ * @import { FormFileUploadStatus, S3FileIdentifier, UploadPayload } from '~/src/api/types.js'
```

Comment:
Unused `S3FileIdentifier`

Reply:
I actually did this but forgot the push 😭